partcad/src/partcad/assembly_factory_assy.py

```
@@ -64,33 +65,24 @@
-                        os.path.dirname(self.path) + os.path.sep
-                    )
-                ).from_string(config)
+                template = Environment(loader=FileSystemLoader(os.path.dirname(self.path) + os.path.sep)).from_string(
+                    config
+                )
                 config = template.render(params)
 
                 # Parse the resulting config
                 try:
                     assy = yaml.safe_load(config)
                 except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)